src/main/java/org/opensearch/knn/index/memory/NativeMemoryAllocation.java

```diff
@@ -132,20 +129,18 @@ class IndexAllocation implements NativeMemoryAllocation {
          * @param memoryAddress Pointer in memory to the index
          * @param sizeKb Size this index consumes in kilobytes
          * @param knnEngine KNNEngine associated with the index allocation
-         * @param indexPath File path to index
+         * @param vectorFileName File path to index
```

Comment:
Is this going to be a path or file name, can you update the name or javadoc to avoid confusion?

Reply:
sure, gave the better branding and also changed java doc!